frontend/Routes.ts

```diff
@@ -212,7 +212,7 @@ export const RouteObj = {
   NamespaceDetails: `${hubRoutePrefix}/namespaces/:id`,
   CreateNamespace: `${hubRoutePrefix}/namespaces/create`,
   EditNamespace: `${hubRoutePrefix}/namespaces/:id/edit`,
-
+  EditNamespaceDetails: `${hubRoutePrefix}/namespaces/:id/edit-details`,
```

Comment:
Is the goal to have a different URL for namespaces edit? I noticed that for the others ones related to hub we are using `${hubRoutePrefix}/item/:id/edit`

Ah, I see what is going, perhaps we could rename to `EditNamespaceMetadataDetails`?